examples/MappingFiles/snomed_to_icd_map.csv

```diff
@@ -0,0 +1,12 @@
+conceptId,mapTarget,ICD10_3digit
+215429004,V909,V90
+704334005,M8024,M80
+429353004,S597,S59
+215063005,V281,V28
+284193003,T215,T21
+721657003,A328,A32
+409701000000100,C791,C79
+584501000000103,L030,L03
+219381006,T1421,T14
+281528007,S5201,S52
+678361000000102,S368,S36
```

Comment:
This is good. For testing purposes, i think it would be easier to just have the test snomed codes we have been using: 
100000001 --> A01
100000002 --> A02 
etc